README.md

```diff
@@ -2,8 +2,8 @@
 
 [![gitleaks badge](https://img.shields.io/badge/protected%20by-gitleaks-blue)](https://github.com/zricethezav/gitleaks#pre-commit) [![gitsecrets](https://img.shields.io/badge/protected%20by-gitsecrets-blue)](https://github.com/awslabs/git-secrets)
 
-Repository for [Configuration as Code for the operations center](https://docs.cloudbees.com/docs/cloudbees-ci/latest/casc-oc/).
+This repository is for [Configuration as Code (CasC) for the operations center](https://docs.cloudbees.com/docs/cloudbees-ci/latest/casc-oc/).
 
-Plugins and plugin catalogs curated with [casc-plugin-dependency-calculation](https://github.com/kyounger/casc-plugin-dependency-calculation).
+Plugins are curated with the [CloudBees CasC Plugin Catalog and Transitive Dependencies Calculator](https://github.com/kyounger/casc-plugin-dependency-calculation).
```

Comment:
@carlosrodlop - I removed the reference to plugin catalogs here, since I don't think they're supported for OC CasC bundles. If this is incorrect, please revert this change.